src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -352,4 +352,6 @@ function editCourse() {
 
 $(document).ready(function() {
     $('#courseEditLink').click(editCourse);
+    var index = $('#new-instructor-index').val();
```

Comment:
With these two lines in close proximity, it also becomes obvious that the [HTML naming convention](https://docs.google.com/document/d/12PJYbQoqjK-0LzaUuguQ4kGE--eikCcHfwzZDGwFOJ0/pub?embedded=true) for IDs is not strictly followed. Most (if not all) elements on this page use camelCase for naming the id of HTML elements.

Considering the other positive examples on the conventions page, it may appear that camelCase was in fact the standard way of naming IDs for awhile. Perhaps this would require more discussion in a separate issue?